modules/nf-core/gatk4/filterintervals/main.nf

```diff
@@ -0,0 +1,47 @@
+process GATK4_FILTERINTERVALS {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::gatk4=4.3.0.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk4:4.3.0.0--py36hdfd78af_0':
+        'quay.io/biocontainers/gatk4:4.3.0.0--py36hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(intervals)
+    path tsv
+    path annotated_intervals
```

Comment:
why no meta for these input?

Reply:
I have only seen meta provided for the input files (`--input`) in other modules. These inputs are optional as well. The meta for the input would be the same for these inputs regardless, so I feel it may be redundant mentioning it again, but perhaps I am wrong?